iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/AbstractLimitCheck.java

```diff
@@ -34,7 +37,15 @@ Class<LimitRange> getGlobalResourceType() {
     return LimitRange.class;
   }
 
+  protected Set<String> getLimitTypes() {
+    return LIMIT_TYPES;
+  }
+
   abstract String getResourceName();
 
   abstract String getMessage();
+
+  static boolean startsWithDigit(@Nullable String value) {
```

Comment:
I moved this method in my PR to `AbstractResourceManagementCheck`, because I also need it in Requests.
I will merge soon.